src/XMLimport.cpp

```diff
@@ -901,42 +914,46 @@ void XMLimport::readHostPackage()
     }
 }
 
-void XMLimport::readHostPackage( Host * pT )
+void XMLimport::readHostPackage( Host * pHost )
 {
-    pT->mAutoClearCommandLineAfterSend = ( attributes().value("autoClearCommandLineAfterSend") == "yes" );
-    pT->mDisableAutoCompletion = ( attributes().value("disableAutoCompletion") == "yes" );
-    pT->mPrintCommand = ( attributes().value("printCommand") == "yes" );
-    pT->set_USE_IRE_DRIVER_BUGFIX( attributes().value("USE_IRE_DRIVER_BUGFIX") == "yes" );
-    pT->mUSE_FORCE_LF_AFTER_PROMPT = ( attributes().value("mUSE_FORCE_LF_AFTER_PROMPT") == "yes" );
-    pT->mUSE_UNIX_EOL = ( attributes().value("mUSE_UNIX_EOL") == "yes" );
-    pT->mNoAntiAlias = ( attributes().value("mNoAntiAlias") == "yes" );
-    pT->mIsNextLogFileInHtmlFormat = ( attributes().value("mRawStreamDump") == "yes" );
-    pT->mAlertOnNewData = ( attributes().value("mAlertOnNewData") == "yes" );
-    pT->mFORCE_NO_COMPRESSION = ( attributes().value("mFORCE_NO_COMPRESSION") == "yes" );
-    pT->mFORCE_GA_OFF = ( attributes().value("mFORCE_GA_OFF") == "yes" );
-    pT->mFORCE_SAVE_ON_EXIT = ( attributes().value("mFORCE_SAVE_ON_EXIT") == "yes" );
-    pT->mEnableGMCP = ( attributes().value("mEnableGMCP") == "yes" );
-    pT->mEnableMSDP = ( attributes().value("mEnableMSDP") == "yes" );
-    pT->mMapStrongHighlight = ( attributes().value("mMapStrongHighlight") == "yes" );
-    pT->mLogStatus = ( attributes().value("mLogStatus") == "yes" );
-    pT->mEnableSpellCheck = ( attributes().value("mEnableSpellCheck") == "yes" );
-    pT->mShowInfo = ( attributes().value("mShowInfo") == "yes" );
-    pT->mAcceptServerGUI = ( attributes().value("mAcceptServerGUI") == "yes" );
-    pT->mMapperUseAntiAlias = ( attributes().value("mMapperUseAntiAlias") == "yes" );
-    pT->mFORCE_MXP_NEGOTIATION_OFF = ( attributes().value("mFORCE_MXP_NEGOTIATION_OFF") == "yes" );
-    pT->mRoomSize = attributes().value("mRoomSize").toString().toInt();
```

Comment:
This is the point of failure all those years ago as mRoomsize is between 0.1 and 1.1 !